webapp/index-self-contained.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>GeometryApp</title>
+
+		<script id="sap-ui-bootstrap"
+			src="resources/sap-ui-custom.js"
+			data-sap-ui-libs="sap.m"
+			data-sap-ui-theme="sap_belize"
+			data-sap-ui-compatVersion="edge"
+			xx-data-sap-ui-preloadLibCss="!sap.ui.core,sap.m"
```

Comment:
this somehow does not work - library css does not load when i check out this change and build